lib/broadway_cloud_pub_sub/google_api_client.ex

```diff
@@ -73,40 +84,88 @@ defmodule BroadwayCloudPubSub.GoogleApiClient do
       opts.subscription.subscriptions_id,
       body: pull_request
     )
-    |> wrap_received_messages(opts.ack_ref)
+    |> wrap_received_messages(opts.ack_ref, opts)
   end
 
   @impl Acknowledger
-  def ack(ack_ref, successful, _failed) do
-    successful
-    |> acknowledge_messages(ack_ref)
+  def ack(ack_ref, successful, failed) do
+    ack_messages(successful, ack_ref, :successful)
+    ack_messages(failed, ack_ref, :failed)
   end
 
-  defp acknowledge_messages([], _), do: :ok
+  @impl Acknowledger
+  def configure(_channel, ack_data, options) do
```

Comment:
This looks like a hold-over from the RabbitMQ Acknowledger, but just to avoid confusion, let's rename the _channel arg

```suggestion
  def configure(_ack_ref, ack_data, options) do
```